src/mesh/RadioInterface.cpp

```diff
@@ -105,6 +105,12 @@ const RegionInfo regions[] = {
         This needs to be last. Same as US.
     */
     RDEF(UNSET, 902.0f, 928.0f, 100, 0, 30, true, false, false)
+        
+    /*
+        868,0-868,6 Mhz 25 mW
+        https://nkrzi.gov.ua/images/upload/256/5810/PDF_UUZ_19_01_2016.pdf
+    */
+    RDEF(UA868, 868.1f, 868.5f, 100, 0, 20, true, false, false)
```

Comment:
Please add the name of the definition to https://github.com/meshtastic/protobufs/blob/e00b5ba7d06053d820f1123351881fc4fa9270d1/config.proto#L506 or else you won't be able to use it. Also please add an underscore in front of the frequency designator to come in line with the other definitions.

Suggestion: UA_868